src/main/resources/application.yml

```diff
@@ -88,6 +88,7 @@ interventions-ui:
     sent-referral: "/service-provider/referrals/{id}/details"
     cancelled-referral: "/service-provider/referrals/{id}/details"
     submit-action-plan: "/service-provider/referrals/{id}/progress" # fixme
+    approved-action-plan: "/service-provider/referrals/{id}/progress" # fixme
```

Comment:
i think we should clarify these urls. for urls stored in delius they must be PP facing URLs. 

for email notifications to SPs they must be SP facing URLs.

etc etc

Reply:
Good point, will make these probation-practitioner